simulation/stress_test/prepare_stress_test.py

```diff
@@ -0,0 +1,111 @@
+import sys
+sys.path.append("../../")
+import os
+import configparser
+import jupiter_config
+import logging
+import random
+import paramiko
+import socket
+import _thread
+import multiprocessing
+from multiprocessing import Process, Manager
+from flask import Flask, request
+
+
+
+logging.basicConfig(level = logging.INFO)
+app = Flask(__name__)
+
+# This script must run on XDC to put random stress test on random nodes in order to collect stats from DCOMP testbed 
+
+def get_nodes(node_info_file):
+  """read the node info from the file input
```

Comment:
Since this is a new file, this is a perfect time to turn on the linter and remove all linting errors. Could you please remove all formatting errors in this file?

Reply:
fixed, except long line